src/lib/wallclock/include.am

```diff
@@ -7,6 +7,7 @@ endif
 
 src_lib_libtor_wallclock_a_SOURCES =			\
 	src/lib/wallclock/approx_time.c			\
+	src/lib/wallclock/tm_cvt.c			\
```

Comment:
I'm not familiar with this naming, what does `tm_cvt` means here?

Reply:
tm is "struct tm"; cvt is convert.